docs/encrypted_images.md

```diff
@@ -69,13 +69,36 @@ but randomizing a 16-byte block with a TRNG should make it highly
 improbable that duplicates ever happen.
 
 To distribute this AES-CTR-128 key, new TLVs were defined. The key can be
-encrypted using either RSA-OAEP or AES-KW-128. Also in the future support
-for EICES (using EC) can be added.
+encrypted using either RSA-OAEP, AES-KW-128 or ECIES.
 
 For RSA-OAEP a new TLV with value `0x30` is added to the image, for
-AES-KW-128 a new TLV with value `0x31` is added to the image. The contents
-of both TLVs are the results of applying the given operations over the
-AES-CTR-128 key.
+AES-KW-128 a new TLV with value `0x31` is added to the image, and for ECIES
+a new TLV with value `0x32` is added. The contents of those TLVs are the
+results of applying the given operations over the AES-CTR-128 key.
+
+## ECIES encryption
+
+ECIES follows a well defined process to generate an encryption key. There are
```

Comment:
I think we shouldn't just call this "ECIES encryption" but perhaps "ECIES keywrap encryption", and then add some text to say that we use this to wrap the key similar to how the other algorithms do it. I'd like to keep things open to someday perhaps be able to use ECIES directly to encrypt the images.